auth-web/src/components/auth/common/Product.vue

```diff
@@ -180,23 +184,26 @@ export default class Product extends Vue {
   get showProductFee () {
     return this.canManageProductFee && this.orgProduct && this.orgProduct.product
   }
-
+  private get isStaff (): boolean {
+    return this.currentUser.roles.includes(Role.Staff)
```

Comment:
This is only for BC Reg staff? not SBC Staff?

Reply:
also should use the account composable (or mixin which we will alter to composable when we update it to compostion-api) here instead of checking roles

Reply:
It is only for BC Reg Staff.
I have updated code to use Mixins